src/main/java/net/mcreator/ui/minecraft/EntityListField.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * MCreator (https://mcreator.net/)
+ * Copyright (C) 2020 Pylo and contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package net.mcreator.ui.minecraft;
+
+import net.mcreator.element.parts.EntityEntry;
+import net.mcreator.minecraft.DataListEntry;
+import net.mcreator.ui.component.JItemListField;
+import net.mcreator.minecraft.ElementUtil;
+import net.mcreator.ui.MCreator;
+import net.mcreator.ui.component.util.PanelUtils;
+
+import javax.swing.*;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class EntityListField extends JItemListField<EntityEntry> {
+
+	private final MCreator mcreator;
+
+	public EntityListField(MCreator mcreator) {
+		this.mcreator = mcreator;
+	}
+
+	@Override protected List<EntityEntry> getElementsToAdd() {
+		JList<String> vlist = new JList<>(ElementUtil.loadAllEntities(mcreator.getWorkspace()).stream()
+				.map(DataListEntry::getName).toArray(String[]::new));
+		int option = JOptionPane.showOptionDialog(mcreator, PanelUtils.northAndCenterElement(
+				new JLabel("Select entities you would like to add to the list:"), new JScrollPane(vlist)),
```

Comment:
These texts (and from other JItemListField implementations) could be externalized. I suggest you do this in a separate PR if you are ready to do this.